Sample/EventSourcing.Sample.Web/Controllers/TransactionController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Domain.Commands;
+using Domain.Queries;
+using EventSourcing.Sample.Tasks.Contracts.Accounts.Commands;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace EventSourcing.Sample.Web.Controllers
+{
+    [Route("api/[controller]")]
+    public class TransactionController : Controller
```

Comment:
I would suggest name controller TransactionsController (so the plural form). Then route will be transactions. Plural form is suggested with the REST api best practices - see this [link](http://www.restapitutorial.com/lessons/restfulresourcenaming.html)